src/App.js

```diff
@@ -53,17 +55,87 @@ const App = () => {
     setResults(workout);
   };
 
+  const [removeExercises, setRemoveExercises] = useState(null);
+  const removeExercise = (workout) => {
```

Comment:
Why do we need a removeExercise function? Why do we not just use the users workouts state and remove the exercise from there? @jandony

Reply:
We can do that too :) How do you remove items from the state?

Reply:
You would do something like this: 

Taking the example of a state being:
`const [array, setArray] = useState([{name: "Greg", age: 2000}, {name:"Jeff", age: 'unknown'}]);
`
Then we could build a function that did something like this:

```
const remove = (workout) => {
const temp = [...array];
const afterRemoval = temp.filter((exercise) => exercise.name !==workout.name);
setArray([...afterRemoval])
}
```
The idea is you want to create a temporary array from the current state, then filter it based on if the passed in workout name or whatever info matches the info in the state, it finds that element, removes it from the array and then you set the array to the new filtered out array.